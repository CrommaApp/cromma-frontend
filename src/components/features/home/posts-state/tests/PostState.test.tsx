import React from 'react';
import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import PostsState from '..';

describe('PostState Component', () => {
	const mockStatement = '로딩 중 입니다.';
	beforeEach(() => {
		render(<PostsState statement={mockStatement} />);
	});

	it('Snapshot', () => {
		const component = renderer.create(<PostsState statement={mockStatement} />);
		expect(component.toJSON()).toMatchSnapshot();
	});

	it('Show statement of fetching post', () => {
		expect(
			screen.getByRole('heading', {
				name: mockStatement,
			}),
		);
	});
});
