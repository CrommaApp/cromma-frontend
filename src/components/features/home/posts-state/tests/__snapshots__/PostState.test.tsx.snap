// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostState Component Snapshot 1`] = `
<div
  className="sc-bdvvtL jCpUOw"
>
  <h1>
    로딩 중 입니다.
  </h1>
</div>
`;
